src/lib/github-service.ts

```
@@ -0,0 +1,355 @@
+
+          await this.request(`/repos/${owner}/${repo}/contents/${file.path}`, {
+            method: 'DELETE',
+            body: JSON.stringify({
+              message: `Delete ${file.path} - ${commitMessage}`,
+              sha: fileData.sha,
+              branch: branch,
+            }),
+          });
+        } else {
+          // Create or update file
+          const body: any = {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)